### PR TITLE
[CoreSpotlight] Fixing breaking change.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/CoreSpotlight/CSCompat.cs [hunk 1]
```diff
@@ -13,6 +14,23 @@ public CSCustomAttributeKey () : this (String.Empty)
...
 		}
 	}
+
+#if !TV
+	public partial class CSSearchQueryContext {
+
+#if !NET
+		[Obsolete ("This property was removed. The getter always returns null and the setter throws and InvalidOperationException.")]
+#else
+		[UnsupportedOSPlatform ("ios16.1")]
```
Comment (rolfbjarne) [inline src/CoreSpotlight/CSCompat.cs:24]: The whole file has a `#if !NET && IOS`, which means this won't be included in .NET.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
.

File: src/CoreSpotlight/CSCompat.cs (hunk 1 above)
Comment (mandel-macaque) [reply src/CoreSpotlight/CSCompat.cs:24]: I'm stupid.

**Review by rolfbjarne — APPROVED** [review]

File: src/CoreSpotlight/CSCompat.cs [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+	public partial class CSSearchQueryContext {
 
+#if !NET
+		[Obsolete ("This property was removed. The getter always returns null and the setter throws and InvalidOperationException.")]
+#else
+		[UnsupportedOSPlatform ("ios16.1")]
+		[UnsupportedOSPlatform ("maccatalyst16.1")]
+		[UnsupportedOSPlatform ("macos13.0")]
 #endif
+		public string[] ProtectionClasses {
```
Comment (rolfbjarne) [inline src/CoreSpotlight/CSCompat.cs:28]: This must be virtual:

Screen Shot 2022-10-17 at 07 16 55

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/CoreSpotlight/CSCompat.cs (hunk 2 above)
Comment (mandel-macaque) [inline src/CoreSpotlight/CSCompat.cs:28]:
```suggestion
		public virtual string[] ProtectionClasses { 
```

**Review by rolfbjarne — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**